extensions/arc/src/ui/dashboards/miaa/miaaConnectionStringsPage.ts

```diff
@@ -0,0 +1,83 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as loc from '../../../localizedConstants';
+import { IconPathHelper, cssStyles } from '../../../constants';
+import { KeyValueContainer, KeyValue, InputKeyValue } from '../../components/keyValueContainer';
+import { DashboardPage } from '../../components/dashboardPage';
+import { ControllerModel } from '../../../models/controllerModel';
+
+export class MiaaConnectionStringsPage extends DashboardPage {
+
+	constructor(modelView: azdata.ModelView, private _controllerModel: ControllerModel) {
+		super(modelView);
+		this.refresh().catch(err => console.error(err));
+	}
+
+	protected async refresh(): Promise<void> {
+		await this._controllerModel.refresh();
+	}
+
+	protected get title(): string {
+		return loc.connectionStrings;
+	}
+
+	protected get id(): string {
+		return 'miaa-connection-strings';
+	}
+
+	protected get icon(): { dark: string; light: string; } {
+		return IconPathHelper.connection;
+	}
+
+	protected get container(): azdata.Component {
+		const root = this.modelView.modelBuilder.divContainer().component();
+		const content = this.modelView.modelBuilder.divContainer().component();
+		root.addItem(content, { CSSStyles: { 'margin': '20px' } });
+
+		content.addItem(this.modelView.modelBuilder.text().withProperties<azdata.TextComponentProperties>({
+			value: loc.connectionStrings,
+			CSSStyles: { ...cssStyles.title }
+		}).component());
+
+		const info = this.modelView.modelBuilder.text().withProperties<azdata.TextComponentProperties>({
+			value: `${loc.selectConnectionString}.&nbsp;`,
+			CSSStyles: { ...cssStyles.text, 'margin-block-start': '0px', 'margin-block-end': '0px' }
+		}).component();
+
+		const link = this.modelView.modelBuilder.hyperlink().withProperties<azdata.HyperlinkComponentProperties>({
+			label: loc.learnAboutPostgresClients,
```

Comment:
Some Postgres specific stuff in this file.  This label and the connection strings themselves.

Reply:
Yeah - this was mostly copy pasted. I haven't done any actual work here yet except get the basic structure laid out. Will send out a separate PR with the corrected values.